tools/osbuilder/node-builder/azure-linux/package_install.sh

```diff
@@ -39,19 +39,6 @@ if [ "${CONF_PODS}" == "yes" ]; then
 	mkdir -p ${PREFIX}/usr/lib/systemd/system/
 	cp -a --backup=numbered src/tardev-snapshotter/tardev-snapshotter.service ${PREFIX}/usr/lib/systemd/system/
 
-	if [ "${SHIM_REDEPLOY_CONFIG}" == "yes" ]; then
```

Comment:
is this change potentially due to a missing rebase? A change was made here recently.